library/zlib.po

```diff
@@ -38,332 +44,490 @@ msgid ""
+"continue pendant la concaténation de plusieurs entrées.  Cet algorithme n'a "
+"aucune garantie cryptographique puissante, et ne doit pas être utiliser ni pour "
+"l'authentification, ni pour des signatures numériques. Conçu comme un algorithme "
+"de somme de contrôle, il n'est pas adapté pour une utilisation sous forme de clé "
+"de hachage générique."
 
 #: ../Doc/library/zlib.rst:44
 msgid ""
-"Always returns an unsigned value. To generate the same numeric value across "
-"all Python versions and platforms, use ``adler32(data) & 0xffffffff``."
+"Always returns an unsigned value. To generate the same numeric value across all "
+"Python versions and platforms, use ``adler32(data) & 0xffffffff``."
 msgstr ""
+"Renvoie systématiquement une value non-signée.  Pour produire systématiquement la "
+"même value à travers toutes les versions de Python et différentes plateformes, "
+"utilisez ``adler32(data) & 0xffffffff``."
 
 #: ../Doc/library/zlib.rst:52
 msgid ""
-"Compresses the bytes in *data*, returning a bytes object containing "
-"compressed data. *level* is an integer from ``0`` to ``9`` or ``-1`` "
-"controlling the level of compression; ``1`` is fastest and produces the "
-"least compression, ``9`` is slowest and produces the most.  ``0`` is no "
-"compression.  The default value is ``-1`` (Z_DEFAULT_COMPRESSION).  "
-"Z_DEFAULT_COMPRESSION represents a default compromise between speed and "
-"compression (currently equivalent to level 6). Raises the :exc:`error` "
-"exception if any error occurs."
-msgstr ""
+"Compresses the bytes in *data*, returning a bytes object containing compressed "
+"data. *level* is an integer from ``0`` to ``9`` or ``-1`` controlling the level "
+"of compression; ``1`` is fastest and produces the least compression, ``9`` is "
+"slowest and produces the most.  ``0`` is no compression.  The default value is "
+"``-1`` (Z_DEFAULT_COMPRESSION).  Z_DEFAULT_COMPRESSION represents a default "
+"compromise between speed and compression (currently equivalent to level 6). "
+"Raises the :exc:`error` exception if any error occurs."
+msgstr ""
+"Il compresse les octets contenus dans *data*, renvoie un objets en octets "
+"représentant les données compressées. Le *niveau*doit être un entier, compris "
+"entre ``0`` et ``9`` ou ``-1`` pour ajuster le niveau de compression ; ``1`` "
+"étant plus rapide et procède une compression légère, ``9`` est plus lente mais "
+"produit la plus forte. ``0`` n'effectue aucune compression. La value par défaut "
+"est ``-1`` (Z_DEFAULT_COMPRESSION). Z_DEFAULT_COMPRESSION offre une value par "
+"défaut proposant un équilibre entre vitesse et taux de compression (actuellement "
+"équivalente à 6). Si une erreur surgit, l'exception :exc:`error` est levée."
 
 #: ../Doc/library/zlib.rst:60
 msgid "*level* can now be used as a keyword parameter."
-msgstr ""
+msgstr "Le *niveau* peut être employé désormais comme un paramètre clé."
```

Comment:
Do not translate the parameter name "level". And keyword parameter looks better translated as "paramètre nommé.".